src/components/common/Dropdown.jsx

```diff
@@ -1,36 +1,146 @@
-import React from 'react';
-
-const Dropdown = () => (
-  <div className="dropdown is-active">
-    <div className="dropdown-trigger">
-      <button className="button" aria-haspopup="true" aria-controls="dropdown-menu">
-        <span>Dropdown button</span>
-        <span className="icon is-small">
-          <i className="fas fa-angle-down" aria-hidden="true"></i>
-        </span>
-      </button>
-    </div>
-    <div className="dropdown-menu" id="dropdown-menu" role="menu">
-      <div className="dropdown-content">
-        <a href="#" className="dropdown-item">
-          Dropdown item
-        </a>
-        <a className="dropdown-item">
-          Other dropdown item
-        </a>
-        <a href="#" className="dropdown-item is-active">
-          Active dropdown item
-        </a>
-        <a href="#" className="dropdown-item">
-          Other dropdown item
-        </a>
-        <hr className="dropdown-divider" />
-        <a href="#" className="dropdown-item">
-          With a divider
-        </a>
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'proptypes';
+import classNames from 'classnames';
+import DropdownItem from './DropdownItem';
+
+const Dropdown = ({
+  id,
+  list,
+  title,
+  handleClick,
+  style,
+  open,
+  hoverable,
+  rightAligned,
+  dropUp,
+}) => {
+  const dropdownNode = useRef();
+  const [isOpen, updateIsOpen] = useState(!!open);
+  const [currentSelection, updateSelection] = useState(title);
+  const dropdownClassName = classNames('dropdown', {
+    'is-active': isOpen,
+    'is-hoverable': hoverable,
+    'is-right': rightAligned,
+    'is-up': dropUp,
+  });
+
+  const dropdownWidth = '350px';
```

Comment:
can this be a prop?

Reply:
Added `width` prop to Dropdown with defaultProp set to 350px.